cf-java-logging-support-log4j2/src/main/java/com/sap/hcp/cf/log4j2/converter/api/Log4jContextFieldSupplier.java

```diff
@@ -6,4 +6,8 @@
 @FunctionalInterface
 public interface Log4jContextFieldSupplier extends EventContextFieldSupplier<LogEvent> {
 
+    int BASE_FIELDS = -100;
+    int CONTEXT_FIELDS = -90;
+    int REQUEST_FIELDS = -80;
```

Comment:
Maybe some explanation somewhere why exactly these values are used here.
This might be hard to find later, if there are issues with orderings.